src/lib.rs

```diff
@@ -745,6 +874,72 @@ fn render_debug_impl(mut w: impl Write, msg: &Message) -> Result<()> {
     Ok(())
 }
 
+fn render_multiplexor_enums(
+    mut w: impl Write,
+    dbc: &DBC,
+    msg: &Message,
+    multiplexor_signal: &Signal,
+) -> Result<()> {
+    ensure!(
+        *multiplexor_signal.multiplexer_indicator() == MultiplexIndicator::Multiplexor,
+        "signal {} is not the multiplexor",
+        multiplexor_signal.name(),
+    );
+
+    let mut multiplexed_signals = HashMap::new();
+    for signal in msg.signals() {
+        if let MultiplexIndicator::MultiplexedSignal(switch_index) = signal.multiplexer_indicator()
+        {
```

Comment:
Wow I'm surprised rustfmt let you get away with the `{` on the next line

Reply:
Cargo fmt actually moves it to the next line ;), just tried (1.50.0).

Reply:
haha, that would make me do `use MultiplexIndicator::*;` :smile: